File: slackapilibrary/src/main/java/jp/co/devjchankchan/slackapilibrary/BootCompletedReceiver.java
package jp.co.devjchankchan.slackapilibrary;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootCompletedReceiver extends BroadcastReceiver {

    public static final String CUSTOM_INTENT = "jp.co.devjchankchan.slackapilibrary.TEST_START";

    @Override
    public void onReceive(Context context, Intent intent) {
        System.out.println("(・∀・) ヤッホー");
        Intent service = new Intent(context, PhysicalBotService.class);
        context.startService(service);
    }
}

File: physicalbotlibrary/src/main/java/jp/co/devjchankchan/physicalbotlibrary/BootCompletedReceiver.kt
package jp.co.devjchankchan.physicalbotlibrary

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootCompletedReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        println("(・∀・) ヤッホー")
        val service = Intent(context, PhysicalBotService::class.java)
        context.startService(service)
    }
}
